tools/bumbershoot/custom_pro_db_annotation_data_manager/data_manager/customProDB_annotation.xml

```diff
@@ -0,0 +1,38 @@
+<tool id="custom_pro_db_annotation_data_manager" name="CustomProDB Annotation" tool_type="manage_data" version="0.0.1">
+    <description>builder</description>
+    <command interpreter="Rscript --vanilla">customProDB_annotation.R
```

Comment:
I'm pretty sure I tried that a while ago and it broke because it couldn't find Rscript. That was before conda though, I'll try again with conda.